toolkits/landcover/run-tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit on error.
+set -e
+
+DIR="toolkits/landcover"
+KEY_FILE_ENC="test/.private-key.json.enc"
```

Comment:
Why is toolkits/landcover/test/.private-key.json.enc being committed to the repo?

Reply:
It's needed for Travis to run integration tests using our service account. Only Travis has the private key needed to unencode the key.